lib/runners/processor/pylint.rb

```diff
@@ -73,6 +73,10 @@ def run_analyzer
 
       trace_writer.message "Analyzing #{files.size} file(s)..."
 
+      if files.size == 0
+        return Results::Success.new(guid: guid, analyzer: analyzer)
+      end
```

Comment:
[suggestion] What about this?

- Fix the position of the message "Analyzing...".
- Add a message for no files found.
- Use `empty?` instead of `size == 0`.

```suggestion
      if files.empty?
        trace_writer.message "No files found."
        return Results::Success.new(guid: guid, analyzer: analyzer)
      end

      trace_writer.message "Analyzing #{files.size} file(s)..."
```